src/lib/commands/setup-draw-command.ts

```diff
@@ -88,12 +88,12 @@ export function setupDrawCommand(
     'vert': vertexShader(coordinator.attributeMapper),
 
     'attributes': {
-      // Corners and uv coords of the rectangle, same for each sprite.
+      // Corners of the rectangle, same for each sprite.
       'vertexCoordinates': [
-        [-0.5, -0.5, 0, 1],
-        [0.5, -0.5, 1, 1],
-        [-0.5, 0.5, 0, 0],
-        [0.5, 0.5, 1, 0],
+        [-0.5, -0.5],  // UV: [0, 1].
+        [0.5, -0.5],   // UV: [1, 1].
+        [-0.5, 0.5],   // UV: [0, 0].
+        [0.5, 0.5],    // UV: [1, 0].
```

Comment:
Previously, the UV coordinates of vertex corners were passed into the shaders. This worked as long as the vertex coordinates didn't move. However, with this PR, the vertex coordinates are pushed out by the vertex shader to make room for borders that extend beyond the shape boundary. Consequently, the UV coordinates must also be changed to match.

Now, rather than passing the UVs into the shaders, they are computed from the vertex coordinates by the fragment shader.